lib/index.js

```diff
@@ -546,7 +550,7 @@ function logEvent(chunk, req, res, options, saveEvent) {
       logData.request.body = req._moBody;
       var parseRequestBodyAsTransferEncodingEndTime = Date.now();
       logMessage(options.debug, 'parseRequestBodyAsTransferEncoding took time ', timeTookInSeconds(parseRequestBodyAsTransferEncodingStartTime, parseRequestBodyAsTransferEncodingEndTime));
-    } else if (req.body) {
+    } else if (req.body && (Number(Buffer.byteLength(JSON.stringify(req.body)) < options.requestMaxBodySizeLimit))) {
```

Comment:
Is there a way to reuse existing variables instead of departing or maybe we can just do char count instead of byte count. This might be a little heavier on cpu since we have to parse to even check the limiter.